### ISSUE TITLE
Path for rsync 3 

### ISSUE THREAD

**Issue by beetlefrosch** [username0x0a/grsyncx#2]
My rsync version 3 is in /usr/local/bin/ 

How can I make grscyncx use this version?

**Comment by username0x0a** [comment]
Is that a `homebrew` version? 🤔 The app currently works on top of the system-provided `rsync` via the fixed `/usr/bin/rsync` path as that one is included with all installations. Is there any benefit of using v3?

**Comment by beetlefrosch** [comment]
Yes, it's the homebrew version. I absolutely need the parameter --iconv!

**Comment by username0x0a** [comment]
Custom command path would be a nice addition then, putting it to the roadmap. 👍

**Comment by beetlefrosch** [comment]
> Custom command path would be a nice addition then, putting it to the roadmap. 👍

Very good! Very good. Thank you.

**Comment by tobsen** [comment]
> Is there any benefit of using v3?

Yes, first of all it is a lot faster and I think there are some changes of the handling of macOS filesystem specials.
As far as I know, Apple won't ship 3, because of the license change.

**Comment by username0x0a** [comment]
In version `0.2`, custom `rsync` path can be used by setting a `RSyncCommandPath` key in app's defaults.

Last time I checked `rsync` 3 from Homebrew it failed to synchronise Finder tags properly so if you have any tips on how to fix this use case with that version, feel free to poke me with more info.

**Comment by username0x0a** [comment]
There were probably many different changes between `v2` and `v3` (arguments in the first place) which need to be researched; further support for `rsync 3` will probably take some time as it might need some UI alterations? I'm not sure about the impact size.

Until then however, you can use the `RSyncCommandPath` default setting mentioned above and put in custom arguments in the _Advanced_ tab of the app. 👍 I've just tested it, `-X` was my greatest concern to make Finder tags working with `v3`.

If you use `rsync 3` more often – shall you provide me with a raw mapping of options between versions 2 ↔ 3 & some further feedback, it would definitely help. 🙌

@beetlefrosch @tobsen
